locale/en.po

```diff
@@ -8699,6 +8699,26 @@ msgstr ""
 msgid "WORLD_RELATIVE_MOVEMENT"
 msgstr "World relative"
 
+msgid "WORLD_SIZE"
+msgstr "World Size"
+
+msgid "WORLD_SIZE_EXPLANATION"
+msgstr ""
+"Changes the number of patches in the world.\n"
```

Comment:
I think this text needs reworking for two reasons: all the other options use a format like `(explanation goes here)` and the second is that the text is two sentences long which is not used by the other explanations. So this should be rewritten as a single sentence and not use a period at the end.

Optionally I think the performance concern could be put in the tooltip, which is *usually* where more in-depth information should be put.